packages/functional-tests/pages/signin.ts

```diff
@@ -74,6 +77,39 @@ export class SigninReactPage extends BaseLayout {
     return this.page.getByRole('button', { name: 'Sign in' });
   }
 
+  get signinBouncedHeading() {
+    return this.page.getByRole('heading', {
+      name: 'Sorry. We’ve locked your account.',
+    });
+  }
+
+  // for backwards compatibility with Backbone
+  // not currently implemented in React, see FXA-8827
+  get permissionsHeading() {
+    return this.page.getByRole('heading', { name: /^Request for permission/ });
+  }
+
+  // for backwards compatibility with Backbone
+  // not currently implemented in React, see FXA-8827
+  get permissionsAcceptButton() {
+    return this.page.getByRole('button', { name: 'Accept' });
+  }
+
+  get signinBouncedCreateAccountButton() {
+    return (
+      this.page
+        .getByRole('button', {
+          name: 'No longer own that email? Create a new account',
+        })
+        // backwards compatibility for Backbone
+        .or(
+          this.page.getByRole('link', {
+            name: 'No longer own that email? Create a new account',
+          })
+        )
+    );
```

Comment:
less verbose 🤷🏻‍♀️ ?
```suggestion
    const name = 'No longer own that email? Create a new account';
    return this.page
      .getByRole('button', { name }) // React
      .or(this.page.getByRole('link', { name })); // Backbone
```